### ISSUE TITLE
Update appears to have broken XFCE's behaviour of displaying HDD/Removable Media.

### ISSUE THREAD

**Issue by zayuim** [void-linux/void-packages#10576]
### System

* xuname:  
```
  dmesg: read kernel buffer failed: No such file or directory
Void 4.19.32_1 x86_64-musl GenuineIntel notuptodate rFFF
```
* package:  
  * Believed to be `gvfs-1.40.0_2`
  * If there is a way to show install history the question could be answered better

### Expected behavior

For XFCE and Thunar to display internal HDDs and removable media on the desktop and under "DEVICES" on Thunar's sidebar.

### Actual behavior

Only "Filesystem root" and "tmp" display as removable drives, and and internal or removable drives do not appear. 

### Steps to reproduce the behavior

`sudo xbps-install -Syu`

**Comment by maxice8** [comment]
i believe the _2 in the 1.40.0 gvfs version was done by me to move gvfsd-google to gvfs-goa to avoid a dependency on gnome-online-accounts.

**Comment by maxice8** [comment]
Can you install the previous version of gvfs in your system ? you can use `xdowngrade` from the `xtools` package to do it just do:

`# xdowngrade /var/cache/xbps/gvfs-1.40.0_1.x86_64.xbps`

Note that you need the previous version of gvfs in your binary cache (it is deleted if you do xbps-remove -O)

**Comment by zayuim** [comment]
@maxice8 Hi! The downgrade worked! Old behavior is restored.

Just want to quickly point out that I am using a musl system. Otherwise everything is now working.

**Comment by maxice8** [comment]
On Thu, Apr 04, 2019 at 06:03:06PM -0700, zayuim wrote:
> @maxice8 Hi! The downgrade worked! Old behavior is restored.
> 
> Just want to quickly point out that I am using a musl system. Otherwise everything is now working.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/issues/10576#issuecomment-480114766

Can you try the new version of gvfs with gnome-online-accounts installed ?

**Comment by zayuim** [comment]
> Can you try the new version of gvfs with gnome-online-accounts installed ?

`Package `gnome-online-accounts' already installed.`

**Comment by maxice8** [comment]
On Thu, Apr 04, 2019 at 06:39:53PM -0700, zayuim wrote:
> > Can you try the new version of gvfs with gnome-online-accounts installed ?
> 
> `Package `gnome-online-accounts' already installed.`
> 
> 
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/issues/10576#issuecomment-480120297

ok so gnome-online-accounts missing is not a problem, something weirdly changed

**Comment by zayuim** [comment]
@maxice8 Unless... Was this installed with the gvfs update, or was this always here before?

**Comment by maxice8** [comment]
On Thu, Apr 04, 2019 at 06:42:36PM -0700, zayuim wrote:
> @maxice8 Unless... Was this installed with the gvfs update, or was this always here before?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/issues/10576#issuecomment-480120691

no the update would remove the dependency of gvfs on gnome-online-accounts, if you install the new gvfs and don't have the gvfs-goa installed you might be able to remove gnome-online-accounts

**Comment by CameronNemo** [comment]
Perhaps gvfs-goa is not being pulled in.

**Comment by maxice8** [comment]
On Thu, Apr 04, 2019 at 09:54:10PM -0700, Cameron Nemo wrote:
> Perhaps gvfs-goa is not being pulled in.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/issues/10576#issuecomment-480147916

can certainly be something i moved from gvfs to gvfs-goa but i only moved (as far as i remember) gvfsd-google and google.mount

**Comment by zayuim** [comment]
Hello I installed gvfs-goa and updated back to the _2 revision, But the same problem persists.

**Comment by b1scu1t** [comment]
@zayuim I requested for `gvfs` to stop having `gnome-online-accounts` or `gvfs-goa` as a hard dependency, as the latter ropes in `webkit2gtk` which has a lot of dependencies (in #10416 ).

I wasn't able to replicate the issue you are facing with `Thunar` or `pcmanfm`. My internal and removable drives read fine in the aforementioned programs. Admittedly, after the update I had to reboot for `Thunar` to mount said drives and to link my phone via MTP (gvfs-mtp). Maybe the gvfs daemon doesn't start automatically or the older pre-update instance is still running?

I had the internal drives listed in `/etc/fstab` (if that's relevant),  to mount automatically and to avoid entering my password on every mount.
